apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -147,9 +149,16 @@ export default {
     visible(expression, columnId) {
       if (expression) {
         try {
-          return Expressions.evaluate(expression, this.internalValues);
+          return executeExpression(
+            expression,
+            this.internalValues,
+            this.tableMetaData
+          );
         } catch (error) {
-          this.errorPerColumn[columnId] = `Invalid visibility expression`;
+          this.errorPerColumn[
+            columnId
+          ] = `Invalid visibility expression, reason: ${error}`;
+          return true;
```

Comment:
should this also remove an error in case of succes?

Reply:
not necessarily because another validation error might exists. However, otherwise good point. Ideally we would clean all errors on every value change and then reevaluate all expressions. Therefore added this clean to the 'watch' on internalvalues.